app/src/main/res/values/strings.xml

```diff
@@ -89,6 +89,8 @@
     <string name="starImage">Favorite star</string>
     <string name="settings">Settings</string>
     <string name="settings_category_title_ui">User interface</string>
+    <string name="settings_category_title_barcode">Barcode</string>
+    <string name="settings_category_title_display">Display</string>
```

Comment:
I'm thinking we should maybe merge these into a single category called "Barcode display".

"Keep screen on" and "Prevent screen lock" only affect the card view and by putting them in a different category from the other "Barcode" options it may confuse users a lot and make them think they affect all screens.

Reply:
Oki, will do that